### ISSUE TITLE
Move Documentation into its own repo & deploy

### ISSUE THREAD

**Issue by xf-** [decentespresso/de1app#103]
The increasing activity in this repo with all the docu commits will not make it easier to find issues or commits with a `bisect`.

I recommend to use something like https://docusaurus.io/ (good example what it can do on its own docu/blog)
- It renders the Markdown to a website
- Can be deployed via github actions as static page onto github or self hosted solution. Every merge into main will de deployed to the website.
- Supports versions and highlighting
- Is FOSS (by facebook)
-  Has a edit button on every page to github and everybody can easily help to improve it

(Only a suggestion)

**Comment by xf-** [comment]
Wiki is not really nice, no versions nothing. search is terrible and you are stuck with GitHub forever, if you want to switch.

**Comment by jeffsf** [comment]
A submodule works for me, as I can use whatever editor I want, as well as be able to associate code and documentation at any point in time. 

I'm a little concerned about "anyone can edit" approaches as Ive found them to be more work to watch for wrong information and have to correct it. There's already enough stale/incorrect information floating around only after a couple years of DE1 experience.

**Comment by xf-** [comment]
Anyone can edit, create a pull request and only merged stuff is online of course. You have a edit button, that takes you to configured URL (e.g. GitHub) and you use the online editor and it will create a fork and a pull request (without any technical knowledge). So if somebody wants do document the `settings.tdb` or fix an typo he can do it.

> There's already enough stale/incorrect information floating around only after a couple years of DE1 experience.

If you have versions attached to an information, you can easily find outdated information like a forum or stack overflow, because it is simply removed in newer documentation version. Nearly all OSS uses some kind of documentation, because it is much easier to refer to right information.

Simple markdown collection has no menu, no search (yes a `grep` works always) and looks like try and error. You can do that also with a tool like docusaurus. It is still simple markdown and a Headline will be a TOC element and every page is in a menu.

**Comment by decentjohn** [comment]
feedback received, but this github is for bugs, so I am closing this.
